### PR TITLE
Use asdf package manager to specify/install erlang and elixir languag…

### PR DESCRIPTION
**Pull request by vraravam**
…es for this project.

Fixes issue #11

### CODE REVIEW

**Comment by vraravam** [comment]
@manusajith - can we get this PR merged please?

File: .tool-versions [hunk 1]
```diff
@@ -0,0 +1,2 @@
+erlang 21.1.1
```
Comment (manusajith) [inline .tool-versions]:
```suggestion
erlang 20.1
```

File: .tool-versions [hunk 2]
```diff
@@ -0,0 +1,2 @@
...
+elixir 1.7.3
```
Comment (manusajith) [inline .tool-versions]:
```suggestion
elixir 1.5.2
```

**Comment by manusajith** [comment]
@vraravam I updated the version of the elixir and erlang that's being used in the deployed version of the site and merged it.

Thanks for sending the PR 🎉

**Pull request merged**